netstandard/ref/System.cs

```diff
@@ -89,7 +90,7 @@ public static partial class AppContext
         public static void SetSwitch(string switchName, bool isEnabled) { }
         public static bool TryGetSwitch(string switchName, out bool isEnabled) { isEnabled = default(bool); throw null; }
     }
-    public sealed partial class AppDomain : System.MarshalByRefObject
+    public partial class AppDomain : System.MarshalByRefObject
```

Comment:
What are the use cases for deriving from `AppDomain`? I don't see any problems with this from the Unity side, I'm just curious.

Reply:
Not sure. @weshaggard might know.

Reply:
This is a bug in .NET Core. AppDomain should have stayed sealed.

Reply:
Agreed I filed https://github.com/dotnet/corefx/issues/31391.

Reply:
I've fixed it here too